package.json

```diff
@@ -1,14 +1,12 @@
 {
   "name": "@openmrs/esm-home",
   "private": true,
-  "husky": {
-    "hooks": {
-      "pre-commit": "pretty-quick --staged && yarn verify && yarn extract-translations"
-    }
-  },
   "workspaces": [
     "packages/*"
   ],
+  "lint-staged": {
```

Comment:
Switched to `lint-staged` from `pretty-quick` because the latter is currently incompatible with Prettier v3+.